src/Factory/ExampleGiftCardFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Factory;
+
+use Setono\SyliusGiftCardPlugin\Model\GiftCardInterface;
+use Sylius\Component\Currency\Context\CurrencyContextInterface;
+
+final class ExampleGiftCardFactory implements ExampleGiftCardFactoryInterface
```

Comment:
Why don't we have a method called `createExample()` on the `GiftCardFactoryInterface`?

Reply:
Because that way, it is clear that you have an ExampleGiftCard.
I think it is cleaner than having just a method in regular factory.